processor/spanmetricsprocessor/config.go

```diff
@@ -20,8 +20,8 @@ import (
 	"go.opentelemetry.io/collector/config"
 )
 
-// Dimension defines the dimension name and optional default value if the Dimension is missing from a span attribute.
-type Dimension struct {
+// KeyValuePair defines the key and optional default value if the key is missing from a span attribute.
+type KeyValuePair struct {
```

Comment:
I think it's only key but not value? How about calling it `DimensionKey`? or `AttributeKey`

Reply:
ah yeah, I'll revert this